src/token/erc721/extensions/erc721_uri_storage.cairo

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo v0.14.0 (token/erc721/extensions/erc721_uri_storage.cairo)
+
+#[starknet::component]
+pub mod ERC721URIstorageComponent {
+    use openzeppelin::introspection::src5::SRC5Component::InternalTrait as SRC5InternalTrait;
+    use openzeppelin::introspection::src5::SRC5Component::SRC5Impl;
+    use openzeppelin::introspection::src5::SRC5Component;
+    use openzeppelin::token::erc721::ERC721Component::InternalImpl as ERC721Impl;
+    use openzeppelin::token::erc721::interface::{IERC721, IERC721Metadata};
+    use openzeppelin::token::erc721::{ERC721Component, ERC721HooksEmptyImpl};
+    use starknet::ContractAddress;
+
+    #[storage]
+    struct Storage {
+        token_uris: LegacyMap<u256, ByteArray>,
+    }
+
+    #[event]
+    #[derive(Drop, PartialEq, starknet::Event)]
+    pub enum Event {
+        MetadataUpdate: MetadataUpdate,
```

Comment:
```suggestion
        MetadataUpdated: MetadataUpdated,
```

For consistency among the codebase we strive to have all (almost) events named in the same way. Following this approach the name of this event should be `MetadataUpdated`, although it is indeed name `MetadataUpdate` in the Solidity counterpart